client_libraries-DEPRECATED/python/src/consumersurveys_v2_discovery.json

```diff
@@ -135,150 +193,122 @@
   "Survey": {
    "id": "Survey",
    "type": "object",
-   "description": "Message defining the Survey collections for the REST API.",
```

Comment:
Similarly with various descriptions removed below. (Is it because this file is generated?)